examples/federated-learning-surface-defect-detection-aggregation.Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM tensorflow/tensorflow:1.15.4
+FROM tensorflow/tensorflow:2.3.0
 
 RUN apt update \
-  && apt install -y libgl1-mesa-glx
+  && apt install -y libgl1-mesa-glx git
 
 COPY ./lib/requirements.txt /home
 
+RUN python -m pip install --upgrade pip
+
 RUN pip install -r /home/requirements.txt
+RUN pip install keras
+RUN pip install tensorflow-datasets
 
-ENV PYTHONPATH "/home/lib"
+ENV PYTHONPATH "/home/lib:/home/plato"
 
 COPY ./lib /home/lib
+RUN git clone https://github.com/TL-System/plato.git /home/plato
```

Comment:
Doesn't seem to need `package` in `plato`. suggest `pip install plato-learn`

Reply:
I will update it until ```Plato``` becomes stable.